CHANGELOG.md

```diff
@@ -19,46 +19,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
- * CI: add actuated-arm64. (#4142)
+ * CI: add actuated-arm64. (#4142, #4252, #4276)
 
 ### Fixed
 
  * cgroup v2: do not set swap to 0 or unlimited when it's not available. (#4188)
  * Set the default value of CpuBurst to nil instead of 0. (#4210, #4211)
- * runc exec: fix setting rlimit_nofile. (#4195, #4265, #4282, #4290)
- * libct/cg/fs: fix setting rt_period vs rt_runtime. (#4094, #4258)
- * script/*: fix gpg usage wrt keyboxd. (#4189)
- * Fix a debug log message for user namespaces in nsexec. (#4311)
  * libct/cg: write unified resources line by line. (#4186)
  * libct.Start: fix locking, do not allow a second container init. (#4271)
  * Fix tests in debian testing (mount_sshfs.bats). (#4245)
- * Fix checkpoint/restore tests on actuated-arm64. (#4276)
  * Fix codespell warnings. (#4291)
  * libct/cg/dev: fix TestSetV1Allow panic. (#4295)
  * tests/int/scheduler: require smp. (#4298)
 
 ### Changed
 
- * libcontainer: allow containers to make apps think fips is enabled/disabled for testing. (#4246)
  * libct/cg/fs: don't write cpu_burst twice on ENOENT. (#4259)
  * Allow overriding VERSION value in Makefile. (#4269)
```

Comment:
This is also present in 1.13, can you delete it too?

It is present on the release notes, though. I think it is fine to delete it and mention in the next 1.2.x release that we removed it as it was included by mistake.

What do you think?

Reply:
Thanks for catching this; PR updated. I've updated release notes as well.

Reply:
Oh, and I think we don't have to tell about after-release changes in the release notes, it will be too much meta :)

We just keep maintaining the changelog and release notes even after the release is made, on a best-effort basis.

Reply:
Perfect!